lib/msal-common/src/utils/Constants.ts

```diff
@@ -3,6 +3,43 @@
  * Licensed under the MIT License.
  */
 
+export const Constants = {
+    // Prefix for all library cache entries
+    CACHE_PREFIX: "msal",
+    // Resource delimiter - used for certain cache entries
+    RESOURCE_DELIM: "|"
```

Comment:
Is this in our code somewhere today? Feels pretty bad to have to manually include a delimiter. I think that just signals to me that entries should be more specific

Reply:
Yes it is in our code today, used by state and cache key entries. I will review if we need to make any changes.